table-api/src/main/java/io/deephaven/api/TableOperations.java

```diff
@@ -517,6 +519,23 @@ TOPS raj(TABLE rightTable, Collection<? extends JoinMatch> columnsToMatch,
 
     // -------------------------------------------------------------------------------------------
 
+    TOPS updateBy(UpdateByClause operation);
+
+    TOPS updateBy(UpdateByClause operation, final String... byColumns);
```

Comment:
I've been changing the order (in another branch) for aggs to keep the no-key variants separate from the variants that have `byColumns`. I think the organization makes more logical sense that way.

Reply:
Done